### DISCUSSION TITLE
Задание 1

### DISCUSSION THREAD

**Discussion by AmastyVadim** [daniil-loban/AmastyTraining#2] [General]
Оценка: 9

https://github.com/daniil-loban/AmastyTraining/blob/master/DaniilLoban/etc/module.xml#L9 - setup_version нам пока не нужен
https://github.com/daniil-loban/AmastyTraining/blob/master/DaniilLoban/composer.json#L14 - неймспейс должен совпадать с именем модуля (`Amasty\\DaniilLoban\\`)

**Comment by daniil-loban** [comment]
Спасибо
